packages/tokens/scripts/extract-carbon-parts/convertObjectToDtcgFormat.ts

```diff
@@ -157,11 +162,27 @@ function recursivelyProcessObject({ key, value, type, group}: Args): CarbonDesig
   }
 }
 
+function convertCubicBezierValue(value: string) {
+  // eg. `cubic-bezier(0.2, 0, 0.38, 0.9)`
+  const match = value.match(/^cubic-bezier\((.*)\)$/);
```

Comment:
This seems a bit broad in what it matches in the parenthesis, but I don't know if that's a real concern.